src/templates/__tests__/LazyTemplateInterface.spec.js

```diff
@@ -0,0 +1,47 @@
+import LazyTemplateInterface from '../LazyTemplateInterface';
+import MockTemplateFactory from './MockTemplateFactory';
+import { constructWithGetters } from '../utils';
+import mockProvider from './mockProvider';
+
+describe('LazyTemplateInterface', () => {
+  it('throws when setting an undefined template', () => {
+    expect(() => constructWithGetters(LazyTemplateInterface, {
+      checkbox: MockTemplateFactory,
+      checkboxGroup: MockTemplateFactory,
+      dateTextField: MockTemplateFactory,
+      dropDown: MockTemplateFactory,
+      formPage: MockTemplateFactory,
+      radio: MockTemplateFactory,
+      staticPage: MockTemplateFactory,
+      struct: MockTemplateFactory,
+      textArea: MockTemplateFactory,
+      textField: MockTemplateFactory,
+
```

Comment:
extra space

Reply:
I put this space and the comment here to make it clearer to the reader that the extra field is extra. Is that not helpful to you as a reader?